src/apis/video.ts

```diff
@@ -31,7 +31,7 @@ export type Video = {
 
 export const getVideoListComplete = async () => {
   const res = await client.get<Video[]>("/videos/completion");
-  return res.data;
+  return res.data.result;
 };
```

Comment:
확인해보니 result 에 대한 타입 지정을 해야된다는 것 같습니다.

두 가지 해결 방법이 있는 것 같습니다.
1. axios return 값에 result 타입을 지정해주기
```suggestion
const res = await  client.post<{ result: Video[] }>(
```

2. type을 result로 감싸기
```suggestion

export type Video = {
result:{
  videoIdx: number;
  link: string;
  title: string;
  admin: string;
  }
};
```

일단 저는 1번 방법으로 하고, 이슈 발생하면 공유하겠습니다